packages/runner-shared/src/iframe/aut-iframe.js

```diff
@@ -69,6 +69,39 @@ export class AutIframe {
     return Cypress.cy.detachDom(this._contents())
   }
 
+  /**
+   * If the AUT is cross origin relative to top, a security error is thrown and the method returns false
+   * If the AUT is cross origin relative to top and chromeWebSecurity is false, origins of the AUT and top need to be compared and returns false
+   * Otherwise, if top and the AUT match origins, the method returns true.
+   * If the AUT origin is 'null', that means the src attribute has been stripped off the iframe and is adhering to same origin policy
+   */
+  doesAUTMatchTopOriginPolicy = () => {
+    const Cypress = eventManager.getCypress()
+
+    if (!Cypress) return
+
+    try {
+      const { location: locationAUT } = this.$iframe[0].contentWindow.document
+      const locationTop = Cypress.utils.locExisting(window.location.href)
+
+      return locationTop.origin === locationAUT.origin || locationAUT.origin === 'null'
```

Comment:
I think here you want to check originPolicy instead of origin, so that you remove the sub domain.

Reply:
Actually this might be ok since you're checking it against top, not the spec bridge, but it wouldn't hurt to check the origin policy.

Reply:
You may be able to use `cors.urlOriginsMatch()` instead.

Reply:
I opted for the `Location` object on big Cypress to create locations in [fba5cf2](https://github.com/cypress-io/cypress/pull/20949/commits/fba5cf275a23008fcd1e20ed5ddfda4509e36225) and compare origin policies. I can also just import the cors from in `@packages/network`  in the `aut-iframe. Is there one we should use over the other?